packages/autorest.python/autorest/codegen/models/lro_operation.py

```diff
@@ -83,6 +83,12 @@ def lro_response(self) -> Optional[LROResponseType]:
             response = responses_with_bodies[0]
         return response
 
+    def response_type_annotation(self, **kwargs) -> str:
+        lro_response = self.lro_response or next(iter(self.responses), None)
```

Comment:
i think in this case the emitter / precheck part of the process should correctly set the lro_response.